WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -807,8 +807,8 @@ extension OrderDetailsDataSource {
 
         let payment: Section = {
             var rows: [Row] = [.payment, .customerPaid]
-            if order.refunds.count > 0 {
-                let refunds = Array<Row>(repeating: .refund, count: order.refunds.count)
+            if !condensedRefunds.isEmpty {
```

Comment:
nit: I believe we also have a helper `isNotEmpty` that might be easier to read 😄

Reply:
Indeed! Thank you, looks better 👍